### PR TITLE
Alha append

### PR DESCRIPTION
**Pull request by alexjhawk**
### Features
- Added function for appending strings to existing files
### Bug Fixes
- N/A
### Other
- N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+### Features
+- Added function for appending strings to existing files
+### Bug Fixes
+- N/A
+### Other
+- N/A
+
+## Version 1.0
+### Features
+- Initial Connector Release
```
Comment (TomKimsey) [inline CHANGELOG.md]: This is not a connector

Comment (TomKimsey) [inline CHANGELOG.md]:
```suggestion
- Initial library release
```

**Review by oliver-walker-hms — DISMISSED** [review]

**Review by TomKimsey — CHANGES_REQUESTED** [review]

**Comment by TomKimsey** [comment]
@alexjhawk  dont forget to tag the repo for v1.0 and v1.1. I would prefer we use some prefix like V or Version in the tag name rather than the tag data lib that just has 1.0 as the tag name.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+### Features
+- Added function for appending strings to existing files
+### Bug Fixes
+- N/A
+### Other
+- N/A
+
+## Version 1.0
+### Features
+- Initial Library Release
```
Comment (TomKimsey) [inline CHANGELOG.md]: why did you make this line all caps where line 5 is not?

Comment (alexjhawk) [reply CHANGELOG.md]: This was a copy/paste and adapt from the Thingworx readme. I've updated so it is uniform.

**Review by TomKimsey — APPROVED** [review]

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**